cfgov/unprocessed/css/molecules/notification.less

```diff
@@ -90,6 +90,7 @@
 
         color: @gray-20;
         font-size: unit( 18px / @base-font-size-px, em );
+        line-height: 1;
```

Comment:
Not sure. The icon was shifted down and all I could find when comparing vs the live site was that `line-height: 1` was set via cf-icon in the live site and wasn't locally. This was the only spot I saw an issue with it.